visualization/app/codeCharta/util/nodeDecorator.ts

```diff
@@ -41,32 +41,37 @@ export class NodeDecorator {
 			}
 		}
 
+		const { nodeMetricData, edgeMetricData } = metricData
 		let id = 0
-
-		const defaultAttributes = { [NodeMetricDataService.UNARY_METRIC]: 1 }
-		const defaultLeafAttributes = { ...defaultAttributes }
-		for (const { name } of metricData) {
-			defaultLeafAttributes[name] = 0
-		}
-
-		for (const { data } of root.descendants()) {
+		hierarchy(map).each(({ data }) => {
```

Comment:
The order does not matter in this case. We might want to check if there's a performance difference for the different methods and which one is the fastest (I guess there might be benchmarks on the net / in the three repo). I personally would also rather use a regular for loop, if possible or to use a named function to get the name in performance profiles.